src/__mocks__/twilio-video.ts

```diff
@@ -1,4 +1,5 @@
 import { EventEmitter } from 'events';
+import { TRACK_TYPE } from '../utils/displayStrings';
 
 class MockRoom extends EventEmitter {
```

Comment:
Let's discuss the concept behind MockRoom? Maybe a comment / documentation here will help.

Reply:
Needs to be familiarised as we continue to familiarise ourselves with the code base